pkgs/tools/compression/lbzip2/default.nix

```diff
@@ -15,6 +15,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoconf automake ];
 
   preConfigure = ''
+    sed -i"" -e 's/AC_PREREQ.*/AC_PREREQ(2.64)/' configure.ac
```

Comment:
`substituteInPlace` would be preferred as it prints warnings when the pattern doesn't match

Reply:
Fixed